### DISCUSSION TITLE
Plotting Pole Figures in MTEX

### DISCUSSION THREAD

**Discussion by aniketchakrabarty** [mtex-toolbox/mtex#550] [Ask Anything] [answered]
[euler_in_degrees.txt](https://github.com/mtex-toolbox/mtex/files/5457847/euler_in_degrees.txt)
Dear Sir,

I have some euler angle data (Bunge Euler Angles phi1 Phi phi2) in the form of a text file.
I am attaching the same.

The data is for 205 grains of Aluminium for some analysis. Each data point (phi1 Phi phi2) is for 1 grain. 

I am using MTEX version 4.5.2 and my MATLAB version is R2014a. So, when i am trying to import the data through the interface, in a way like this :

![image](https://user-images.githubusercontent.com/73643457/97559720-cfe44600-1a03-11eb-8d5a-9d4b603d19d4.png)

certain error is appearing at the time, when 'FINISH' is clicked. It is as follows

![image](https://user-images.githubusercontent.com/73643457/97559838-f4d8b900-1a03-11eb-90f9-f8cc520e6bcf.png)

I want to simply plot the ODF and the 100, 011 and 111 Pole Figures for the data. Also, I want to plot the ODF. Could you please help me with plotting this data?

**Comment by ralfHielscher** [comment, marked as answer]
Hi,

1. I think it would be better for you to update to a more recent Matlab version  + a more recent MTEX version
2. You are not importing EBSD data but orientations. Hence, you should do something like
```matlab
cs = crystalSymmetry('cubic')
ori = loadOrientation_generic('euler_angles.txt.','CS',cs,'ColumnNames', {'Euler1' 'Euler2' 'Euler3'})
```
Ralf.

**Reply by aniketchakrabarty** [reply]
Dear Sir, 

Thank you. This works !!!
